dotnet/src/Extensions/PromptTemplates.Liquid/LiquidPromptTemplate.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Scriban;
+
+namespace Microsoft.SemanticKernel.PromptTemplates.Liquid;
+
+internal sealed class LiquidPromptTemplate : IPromptTemplate
```

Comment:
Please add XML comments to the class and its public members. Do the same for the other classes in the PR.

Reply:
track in #6033